.editorconfig

```diff
@@ -0,0 +1,15 @@
+# Declares that this is the top-level configuration
```

Comment:
please add .editorconfig to .gitignore too, as it is per developer to setup own

Reply:
Done

Reply:
Note this now means that the default tabsize when viewing in github is 8, which was what I was initially trying to correct